release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -344,6 +344,13 @@ The following related APIs will be removed in a future release:
 [id="ocp-4-4-bug-fixes"]
 == Bug fixes
 
+*Web Console*
+
+* Earlier, when two consecutive rollouts failed, the *Topology* view showed failed pods instead of displaying the last active revision. With this bug fix, when a rollout fails the last active revision is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
+
+* Previously, when a user with limited cluster-wide permissions created an application using the *Container Image* option in the *Add* page, and chose the *Image name from the internal registry* option, no imagestreams were detected in the project, though an imagestream existed. Now, the imagestream fetching logic has been moved from the cluster level to the namespace level enabling an user with permission to a namespace to see imagestreams in that namespace.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1784264[*BZ#1784264*])
```

Comment:
s/using/by using

s/and,/and

The first sentence is too long.

s/level enabling/level, which enables

Reply:
@kalexand-rh I have made the changes and have tried to shorten the sentence, please let me know if this is alright.